src/fusor/fusion_caller_models.py

```diff
@@ -4,6 +4,7 @@
 from enum import Enum
 from typing import Literal
 
+from civicpy.civic import CivicAttribute
```

Comment:
Whenever you introduce a third-party package, you need to add this to the dependency list in pyproject.toml